src/pfe/portal/modules/Project.js

```diff
@@ -25,9 +25,6 @@ const metricsService = require('./metricsService');
 
 const log = new Logger(__filename);
 
-// Declaring STATES here makes it accessible in this file.
-// We add it to the exported Project class so it can be
-// reached by users of the Project class.
```

Comment:
Why remove this comment? It is still exported here: https://github.com/eclipse/codewind/blob/0191c5b6c13d7687010ba016e96fef3b53e4cec5/src/pfe/portal/modules/Project.js#L830
(We don't have the same comment on CW_SETTINGS_PROPERTIES so I know it's not the most consistent commenting but I'm just not sure why this was deleted.)